semgrep-core/matching/Generic_vs_generic.ml

```diff
@@ -1298,13 +1298,25 @@ and m_list__m_attribute (xsa: A.attribute list) (xsb: A.attribute list) =
       with Not_found -> fail ()
       )
   (*e: [[Generic_vs_generic.m_list__m_attribute]] [[NamedAttr]] pattern case *)
-  (* the general case *)
```

Comment:
Note that "the general case" is actually just the specific "OtherAttribute" case